src/omnibar/omnibar-user-session-expiration.spec.ts

```diff
@@ -38,6 +42,7 @@ describe('Omnibar user session expiration', () => {
     )
       .then((expirationDate) => {
         expect(expirationDate).toBeNull();
+        expect(domainSpy).toHaveBeenCalled();
```

Comment:
These should be changed to check the result of having called `getSTSDomain()` rather than simply if it was called, e.g. check to see if the return value from `getSTSDomain()` was used in subsequent logic like making a web request to a URL.